### PR TITLE
A bit modification of about form

### PR DESCRIPTION
**Pull request by T3RY4**
Here I tried to edit about form more stylish. So whats do you think?

### CODE REVIEW

File: ShareX/Forms/AboutForm.cs [hunk 1]
```diff
@@ -27,13 +27,32 @@
...
+using System.Runtime.InteropServices;
 
 namespace ShareX
 {
     public partial class AboutForm : Form
     {
         private EasterEggAboutAnimation easterEgg;
 
+        [DllImportAttribute("user32.dll")]
+        public static extern int SendMessage(IntPtr hWnd, int Msg, int wParam, int LPAR);
```
Comment (sylveon) [inline ShareX/Forms/AboutForm.cs]: Use the NativeMethods class.

**Comment by T3RY4** [comment]
Screenshot of modification about form:[https://imgur.com/84irODE](url)

**Comment by T3RY4** [comment]
I tested and now I cant find any bugs with this feature.

**Comment by Jaex** [comment]
Sorry but I prefer to keep title bar of that form for consistency with other forms.

**Pull request closed without merging**